src/common/config_opts.h

```diff
@@ -270,7 +270,7 @@ OPTION(mon_pg_check_down_all_threshold, OPT_FLOAT, .5) // threshold of down osds
 OPTION(mon_cache_target_full_warn_ratio, OPT_FLOAT, .66) // position between pool cache_target_full and max where we start warning
 OPTION(mon_osd_full_ratio, OPT_FLOAT, .95) // what % full makes an OSD "full"
 OPTION(mon_osd_nearfull_ratio, OPT_FLOAT, .85) // what % full makes an OSD near full
-OPTION(mon_allow_pool_delete, OPT_BOOL, true) // allow pool deletion
+OPTION(mon_allow_pool_delete, OPT_BOOL, false) // allow pool deletion
```

Comment:
@wido we might want to update https://github.com/ceph/teuthology/blob/master/teuthology/ceph.conf.template before merging this change. because quite a few of our qa suites are deleting pools without setting `mon_allow_pool_delete`.